lib/collection/request-body.js

```diff
@@ -71,6 +77,9 @@ _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
         // If mode is urlencoded but options does not provide any content, set it to an empty property list
         (mode === RequestBody.MODES.formdata && !formdata) && (formdata = new PropertyList(FormParam, this, []));
 
+        // Set protocolProfileBehavior to undefined for unknown behaviors
+        !_.includes(PROTOCOL_PROFILE_BEHAVIORS, protocolProfileBehavior) && (protocolProfileBehavior = undefined);
```

Comment:
Do a strict equality check for now since we haven’t decided how it will look like.

This is a temporary job. The full work on this will need the following:

- decision on how to store multiple options and if even they have value then it may need to be object 
- will this also club behaviour such as response 3xx behaviour 
- will ssl strict check be here too?

As such, just keep this abstring check for now or an object with key as disableBodyPruning and value as `*` (meaning for all methods)